phylogenetic/rules/export.smk

```diff
@@ -30,7 +31,7 @@ rule export:
             --tree {input.tree} \
             --metadata {input.metadata} \
             --metadata-id-columns {params.strain_id} \
-            --node-data {input.branch_lengths} {input.nt_muts} {input.aa_muts} \
+            --node-data {input.branch_lengths} {input.nt_muts} {input.aa_muts} {input.year} \
```

Comment:
_not asking for a change, just curious_

Instead of making another node-data JSON, why not just add a `year` column to the metadata.tsv that can be used directly as a coloring?

Reply:
I found the approach here cleaner to implement, although if other rules used the data I'd have probably gone for modifying the TSV. It should also be trivial to copy & paste into other workflows which would benefit from this. Of course, once Auspice has a proper temporal scale this'll all be removed anyways.

Reply:
Perhaps analogous to avian-flu where we currently add in a column (H5 LABEL) to the TSV but want to [switch away from this to a node-data JSON](https://github.com/nextstrain/avian-flu/issues/25)